### PR TITLE
Update Game End Canvas #22

### PR DESCRIPTION
**Pull request by sidhanshamil**

### CODE REVIEW

**Comment by shubhansu-kr** [comment]
@sidhanshamil Please add basic description for the pull request. #22

**Comment by shubhansu-kr** [comment]
What was the result of your change ?  Can we get a screenshot of the game ?

**Comment by sidhanshamil** [comment]
Screenshot from 2024-10-02 01-33-17
Screenshot from 2024-10-02 01-33-59

I have attached a screenshot of the changes made. Could you please provide your feedback on whether this is acceptable?

**Comment by shubhansu-kr** [comment]
Clicking on continue will restart the game, right ? If yes, Can we rename the button to Restart Game.

**Comment by sidhanshamil** [comment]
Yes, it will restart the game. I will rename the button accordingly

File: game.js [hunk 1]
```diff
@@ -227,18 +227,45 @@ function drawEverything() {
...
+    const buttonHeight = 60;
+    const buttonX = canvasWidth / 2 - buttonWidth / 2;
+    const buttonY = canvasHeight / 2 + 50;
+
+    canvasContext.fillStyle = isDarkMode ? "#444" : "#ddd";
+    canvasContext.fillRect(buttonX, buttonY, buttonWidth, buttonHeight);
+
+    canvasContext.fillStyle = isDarkMode ? "white" : "black";
+    canvasContext.font = "24px Orbitron";
+    canvasContext.fillText("Continue", canvasWidth / 2, buttonY + buttonHeight / 2);
```
Comment (shubhansu-kr) [inline game.js]: Rename `Continue` to `Restart Game`

Comment (sidhanshamil) [reply game.js]: Sure, I will make that change

**Comment by shubhansu-kr** [comment]
What's with the incorrect commit ?

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Security Policy
```
Comment (shubhansu-kr) [inline SECURITY.md]: This is assigned to someone else i believe here #2

Comment (sidhanshamil) [reply SECURITY.md]: I'm sorry about that. Just a moment, and I can resolve this

**Pull request closed without merging**